spec/rest-api/ddl-service.yaml

```diff
@@ -0,0 +1,500 @@
+openapi: 3.0.0
+
+servers:
+  - url: http://{host}:{port}/dae/v1/ddl
+info:
+  version: "1.0.0"
+  title: "DDL service"
+  description: 'Rest API for DDL service, including CRUD operation on tables (tabular table, rawstore table) and columns.'
+paths:
+  /tables:
+    post:
+      summary: Create a tabular table
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/tabularTable'
+      parameters:
+        - name: x-bdg-request-id
```

Comment:
What about `username` ?

Reply:
The following params are added: X-ClickHouse-Query-Id X-ClickHouse-User X-ClickHouse-Key X-ClickHouse-Quota user password and quota_key